doc/experiment.rst

```diff
@@ -1,326 +1,350 @@
-  # each of the implementations of RunSettings
-  srun.set_nodes(1)
-  srun.set_tasks(32)
-
-  model = exp.create_model("hello_world", srun)
-  exp.start(model, block=True, summary=True)
-
-  print(exp.get_status(model))
-
-The above will run ``srun -n 32 -N 1 echo Hello World!``, monitor its
-execution, and inform the user when it is completed. This driver script can be
-executed in an interactive allocation, or placed into a batch script as follows:
-
-.. code-block:: bash
-
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
-
-    python /path/to/script.py
-
+:ref:`Model(s)<model_object_doc>` represent a simulation model or any computational kernel,
+including applications, scripts, or generally, a program. They can
+interact with other SmartSim entities via data transmitted to/from
+SmartSim ``Orchestrators`` using a SmartRedis ``Client``.
+
+A ``Model`` is created through the factory method: ``Experiment.create_model()``.
+``Models`` are initialized with ``RunSettings`` objects that specify
+how a ``Model`` should be launched via a workload manager
+(e.g., Slurm) and the compute resources required.
+Optionally, the user may also specify a ``BatchSettings`` object if
+the model should be launched as a batch job on the WLM system.
+The ``create_model()`` factory method returns an initialized ``Model`` object that
+gives you access to functions associated with the :ref:`Model API<model_api>`.
+
+A ``Model`` supports key features, including methods to:
+
+- :ref:`Attach configuration files<files_doc>` for use at ``Model`` runtime.
+- :ref:`Colocate an Orchestrator<colo_model_doc>` to a SmartSim ``Model``.
+- :ref:`Attach a ML model<ai_model_doc>`  into the ``Orchestrator`` at ``Model`` runtime.
+- :ref:`Attach a TorchScript function<TS_doc>`  into the ``Orchestrator`` at ``Model`` runtime.
+- :ref:`Enable SmartSim Model data collision prevention<model_key_collision>`, which allows for reuse of key names.
+
+Visit the respective links for more information on each topic.
+
+.. _ensemble_exp_docs:
 Ensemble
 ========
+In addition to a single ``Model``, SmartSim allows users to create,
+configure, and launch an :ref:`Ensemble<ensemble_doc>` of ``Model`` objects.
+``Ensembles`` can be given parameters and permutation strategies that define how the
+``Ensemble`` will create the underlying ``Model`` objects. Users may also
+manually create and append ``Model(s)`` to an ``Ensemble``. For information
+and examples on ``Ensemble`` creation strategies, visit the :ref:`Initialization<init_ensemble_strategies>`
+section within the :ref:`Ensemble<ensemble_doc>` documentation.
+
+An ``Ensemble`` supports key features, including methods to:
+
+- :ref:`Attach configuration files<attach_files_ensemble>` for use at ``Ensemble`` runtime.
+- :ref:`Load AI models<ai_model_ensemble_doc>` (TF, TF-lite, PT, or ONNX) into the ``Orchestrator`` at ``Ensemble`` runtime.
```

Comment:
Very small critque, but we say "Load AI Models" and "Load TorchScripts" in the ensemble section but "Attach a ML model" and "Attach a Torchscript function" in the ``Model`` section.  Maybe we can use the same wording.  Would "Load ML Models" and "Load TorchScripts" work?

Reply:
I like load better!